rsocket/internal/SetupResumeAcceptor.cpp

```diff
@@ -197,61 +172,59 @@ void SetupResumeAcceptor::accept(
   auto processor = std::make_shared<OneFrameProcessor>(
       *this, transport, std::move(onSetup), std::move(onResume));
   connections_.insert(transport);
-  // transport can receive frames right away
+  // Transport can receive frames right away.
   transport->setFrameProcessor(std::move(processor));
 }
 
-std::shared_ptr<FrameSerializer>
-SetupResumeAcceptor::getOrAutodetectFrameSerializer(
-    const folly::IOBuf& firstFrame) {
-  if (defaultFrameSerializer_) {
-    return defaultFrameSerializer_;
+std::shared_ptr<FrameSerializer> SetupResumeAcceptor::frameSerializer(
```

Comment:
Fair enough, I can see how `frameSerializer()` might be construed as being a simple getter.  Can I convince you on 'createSerializer()`?  I like keeping method names short, and the `create` part suggests that there's some expensive work being done.  Plus the method has a comment saying that it'll do the autodetect logic.